test/Microsoft.Sbom.Targets.E2E.Tests/Microsoft.Sbom.Targets.E2E.Tests.csproj

```diff
@@ -0,0 +1,63 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net472</TargetFrameworks>
```

Comment:
This project has been a bit tricky to work with. When targeting net8.0, we were getting assembly issues where it could not find the correct version of the NuGet.Frameworks assembly:

 error MSB4018: The "ProcessFrameworkReferences" task failed unexpectedly.
C:\Program Files\dotnet\sdk\8.0.400-preview.0.24324.5\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(90,5): error MSB4018: System.IO.FileLoadException: Could not load file or assembly 'NuGet.Frameworks, Version=6.11.0.105, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. Could not find or load a specific file. (0x80131621)